### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by curtcorum**
Added github URL to bibtex entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ If you use the package, please cite:
   author = {M. J. Muckley and R. Stern and T. Murrell and F. Knoll},
   title = {{TorchKbNufft}: A High-Level, Hardware-Agnostic Non-Uniform Fast {Fourier} Transform},
   booktitle = {ISMRM Workshop on Data Sampling \& Image Reconstruction},
-  year = 2020
+  year = 2020,
+  url = {https://github.com/mmuckley/torchkbnufft}
```
Comment (mmuckley) [inline README.md:232]:
```suggestion
  note = {Source code available at https://github.com/mmuckley/torchkbnufft}.
```

**Review by mmuckley — CHANGES_REQUESTED** [review]
Requesting a slight tweak for this. I think typically the URL field should point to the publication rather than the source code, so we should change this to a note. What do you think?

File: README.md (hunk 1 above)
Comment (curtcorum) [reply README.md:232]: Looks good to me. The Workshop posters are unfortunately not available, even after an embargo, except to that year's attendees.

**Review by curtcorum — COMMENTED** [review]
Looks great...

**Comment by mmuckley** [comment]
Hello @curtcorum, I need you to commit my suggestion :). Then I can merge.

**Comment by curtcorum** [comment]
Ok, did not hit the button last time!

**Review by mmuckley — APPROVED** [review]
LGTM.

**Pull request merged**